### DISCUSSION TITLE
RIA Narrative

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#13] [Ideas]
## About RIA
The Relay Improvement Association of Baltimore County, Inc. (RIA) is a Maryland non-profit, non-stock, non-sectarian, non-political organization whose purpose is to promote the improvement and general welfare of the neighborhoods within the following boundaries. Boundaries are the Patapsco River to the South, Route 1 and Herbert Run to the East, I-95 to the North, and Selford Avenue above I-95 to the West, including homes on both sides of Gun Road to the State Park property line

## {Include the map?}
<img width="513" alt="image" src="https://github.com/Relay-Maryland/relaymaryland.com/assets/11720131/29bb0731-f27c-4664-a8ed-8fc331e44eea">

## RIA membership

An RIA Member is any person over eighteen (18) years of age (i) residing within the boundaries defined or (ii) owning real property within such boundaries, and who has paid dues to the Association for the current year.
Associate RIA membership is available to those who reside outside of the boundaries of the association. Associate Members will receive newsletters and member discount on RIA events. Associate RIA Members are not counted in Quorum and may not vote. RIA membership dues are paid annually. {add info or backlink to payment options?}

## RIA General Meetings
Meetings are held at the Town Hall at 7:30 p.m. on the second Monday of February, May, September and November
As a member of the RIA, you are encouraged to attend general meetings. These meetings keep you informed of what’s happening in our neighborhood and give you an opportunity to socialize with other community residents.  The RIA welcomes you to read our newsletters and check out our upcoming events and announcements.

## RIA Officers
The RIA Officers consist of a President, Vice-President, Secretary, and Treasurer. Officers are elected by a majority vote at the General Meeting in years ending in odd numbers and shall serve a two-year term. Each elected Officer must have resided within the boundaries of the Association and have been a Member for not less than one year prior to elections.

### Current RIA Officers:
President: Frank Meek
Vice President: Casey Jackson
Treasurer: Barbara McClung
Secretary: Joe Sherren

## Additional Benefits of an RIA membership
Quarterly community newsletter sent to your email
Reduced rental fee to the Historic Town Hall ($125.00 value)
Assisting the RIA with repairs on our historic landmarks & hall
Voting rights on issues affecting Historic Relay
Historic House plaques

@brianzelip

**Comment by brianzelip** [comment]
Done
